src/stochastic_functions/propose.jl

```diff
@@ -28,12 +26,13 @@ The return function is a function that produces a list with all the subprograms
 # Arguments
 - `enumeration_depth::Int64`: the maximum enumeration depth.
 """
+# TODO: Refactor to not return functions
+# TODO: Update documentation with correct function signature
```

Comment:
Please add these (and other TODOs introduced) as GitHub issues and remove them from the source code—that way, we can keep track of progress/assign them to people as needed.

Reply:
Ok I will do that.

Reply:
I added the issues #88 #87